charts/jenkins/values.yaml

```diff
@@ -476,6 +476,8 @@ master:
     # See here: https://prometheus.io/docs/prometheus/latest/configuration/alerting_rules/
     # The `groups` root object is added by default, simply add the rule entries
     alertingrules: []
+    # Set a custom namespace where to deploy PrometheusRule resource
+    # prometheusRuleNamespace: monitoring
```

Comment:
Can we uncomment it and use empty string as default? That enables auto completion in some IDEs.

Reply:
@torstenwalter i changed it as you suggested. please check now

Reply:
Looks good to me. Can you merge main into this branch and fix the merge conflicts?

Reply:
done